ValveResourceFormat/IO/MapExtract.cs

```diff
@@ -13,14 +15,34 @@
 
 namespace ValveResourceFormat.IO;
 
+public class VmapContentFile : ContentFile
+{
+    public string SkyboxPath { get; init; }
+}
+
+[Flags]
+public enum VmapOptions
+{
+    None = 0,
+    Export3DSkybox = 1,
+}
```

Comment:
Check out ShaderExtract.cs and some others for better idea of Options class (called Params in shaderextract).

It will inevitably be changed to a struct so better to do it now `struct VmapOptions { bool Export3DSkybox {get;set;}}`

Reply:
Another example option would be vmap version. Couldn't elegantly have that in flags.

Reply:
why are enum flags a bad idea tho? just genuinely curious

Reply:
Works for this scenario, but not easily extendable.